server/apiController.js

```diff
@@ -118,3 +120,76 @@ module.exports.getTrailer = (req, res) => {
     .then(results => res.send(results.data.items[0]))
     .catch(err => res.sendStatus(404));
 };
+
+module.exports.getSearchAutoComplete = (req, res) => {
```

Comment:
That's a great point, but I don't know how to hide API keys from the user if those requests need to live on the front end. Any tips?